webservices/filters.py

```diff
@@ -17,6 +17,15 @@ def parse_exclude_arg(arg):
     else:
         return arg[1:]
 
+def build_exclude_list(value_list):
```

Comment:
Rather than calling this all the time, even without a value, 
I think we should add back in `if kwargs.get(key):` before parsing the values. Also, it might be slightly more effective to do one function like this:
```
def split_list_args(value_list):
    exclude_list = [parse_exclude_arg(value) for value in value_list if is_exclude_arg(value)]
    include_list = [value for value in value_list if not is_exclude_arg(value)]
    return exclude_list, include_list
```
and then
``exclude_list, include_list = split_list_args(kwargs.get(key))``